packages/plugins/puppetlabs-stdlib/puppetlabs-stdlib.spec

```diff
@@ -0,0 +1,76 @@
+%global commit 45454b8de4ff8f860f6f78438107133e510336ed
+%global shortcommit %(c=%{commit}; echo ${c:0:7})
+
+Name:		puppetlabs-stdlib
+Version:	4.25.1
```

Comment:
I'd prefer not to ship such an old version.

Reply:
We can update to 5.2 since latest foreman_scap_client requires `< 6`. https://github.com/theforeman/puppet-foreman_scap_client/blob/master/metadata.json#L42

If this starts a discussion of updating foreman_scap_client I am going to vote instead to take this as is to match EL7 and then discuss updating this further.

Reply:
https://github.com/theforeman/puppet-foreman_scap_client/pull/61